two questions answered (#31)
did-app/www: faq/index.md

In this case your device could be accessed by someone other than you.  Your devices should always be locked either with a pin code or biometric key such as your fingerprint or face to prevent thieves accessing any of your device’s data.

As an additional security step you can remove authorisation for the lost or stolen device inside your DID.app account (this feature is currently being developed).

<a name="what-happens-if-i-want-to-sell-my-device"></a>
### What happens if I want to sell my device?

Before you sell any device you would need to remove all of your personal data including any keys used by DID.app to authenticate that device.

If you forget to remove all your data from the device when you sell it you can remove authorisation for that device inside your DID.app account at any time (this feature is currently in development).

<a name="how-is-authentication-via-email-secured"></a>
### How is account recovery via email secured?

A link containing a single use code is generated and emailed to the user.
As only the owner of the email address can receive the link only they can access the account.

> ### Where are the private keys for my device stored.

Reviewer: ```suggestion
### Where are the private keys for my device stored?
```

> Key pairs are generated and stored by the browser.

Reviewer: ```suggestion
Key pairs are generated and stored by your browser.
```

From within the browser it is not possible to access the private key, only a reference to a private key.
Private key references are stored in [indexeddb](https://developer.mozilla.org/en-US/docs/Web/API/IndexedDB_API/Using_IndexedDB),

> To see all the key references stored select the storage tab in web developer tools.

Reviewer: ```suggestion
To see all the key references stored, select the storage tab in web developer tools.
```

<a name="why-wouldnt-i-just-use-social-logins"></a>
### Why wouldn't I just use social logins?
DID.app is more convenient to use than social login.  Users that do not have Facebook or Google accounts cannot use those social logins.  If the user does have an account with Facebook or Google they still need to log into it using an email address and password.

Login method choice can cause confusion for the user.  Some confusion can also occur if a user has previously signed up with Facebook but now tries to login with Google or vice versa.

There is also a security threat to consider.  If a Google or Facebook account is compromised the hacker now has access to all the websites which the social login has been used to grant access.  

Some users are also aware of the privacy concerns around the business models of Facebook and Google and their motivations for providing login functionality for tracking purposes.  

DID.app offers a simple, secure solution that is easier to use than social logins and respects the user’s privacy.

### Why does DID.app send a magic link rather than a code?

> When copying a code it is possible for an attacker to mislead a user in to copying a code into the attackers website.

Reviewer: ```suggestion
When copying a code it is possible for an attacker to mislead a user in to copying a code into the attacker's website.
```

> This enables the attacker to compromise the users account.

Reviewer: ```suggestion
This would enable the attacker to compromise the user's account.
```

Requiring users to click on a link is more secure than asking them to copy a code.

> The link includes the correct domain and is more resistant to phishing attack described above.

Reviewer: ```suggestion
The link includes the correct domain and is more resistant to kind of phishing attack described above.
```

Author: @richardesigns this doesn't quite make sense to me

Reviewer: @CrowdHailer You're right, I missed the 'the'.

Author: Ah yes, thanks.
